### PR TITLE
cater for the fact that the qa and sb server environments have been u…

### PR DESCRIPTION
**Pull request by philippe-leonhardt-cko**
…pdated to provide fallback to environment variables if checkout api configuration is omitted in appsettings.json

### CODE REVIEW

File: CKODemoShop/Checkout/CheckoutApiFactory.cs [hunk 1]
```diff
@@ -8,12 +8,10 @@ public static class CheckoutApiFactory
     {
         public static CheckoutApi ConfiguredFromEnvironment()
         {
-            var liveMode = false;
-            bool.TryParse(Environment.GetEnvironmentVariable("CKO_LIVE_MODE"), out liveMode);
             var api = CheckoutApi.Create(
-               secretKey: Environment.GetEnvironmentVariable("CKO_SECRET_KEY"),
-               publicKey: Environment.GetEnvironmentVariable("CKO_PUBLIC_KEY"),
-               useSandbox: !liveMode
+               secretKey: Environment.GetEnvironmentVariable("CHECKOUTAPIOPTIONS__SECRETKEY"),
```
Comment (daniel-munch-cko) [inline CKODemoShop/Checkout/CheckoutApiFactory.cs]: Actually, I think the `ConfiguredFromEnvironment` is redundant, as the asp.net core config mechanism reads these variables already by default when using the other config mechanism. This is where the confusion might be coming from.

Comment (philippe-leonhardt-cko) [reply CKODemoShop/Checkout/CheckoutApiFactory.cs]: `ConfiguredFromEnvironment` 🗑

**Comment by philippe-leonhardt-cko** [comment]
Maybe it makes sense to keep `ConfiguredFromEnvironment` after all?
Tests are failing now, because they were depending on it
https://github.com/CKOTech/checkout-web-payment-demo/blob/master/CKODemoShop.Tests/CheckoutTests.cs#L28

**Comment by daniel-munch-cko** [comment]
if you only need it for the tests then it's an indication that you might want to have it only in the test project. also, I don't

**Pull request merged**